packages/Swiper/styles.js

```diff
@@ -8,12 +8,12 @@ export const Wrapper = styled.div`
 `
 
 export const Swiper = styled.ul(
-  ({ translateX }) => css`
+  ({ slidesToShow, translateX }) => css`
     list-style: none;
     margin: 0;
     padding: 0;
     display: flex;
-    width: 100%;
+    width: ${() => `${100 / slidesToShow}%`};
```

Comment:
Does this need to be in a function? Wouldn't this works as well?
```suggestion
    width: ${100 / slidesToShow}%};
```